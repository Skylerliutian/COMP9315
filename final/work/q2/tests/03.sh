./bnl 5 4 10
