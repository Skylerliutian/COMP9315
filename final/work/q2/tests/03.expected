Running: ./bnl 5 4 10

Frames:     [00] [01] [02] [03] [04] [05] [06] [07] [08] [09]
Contents:    R00  S00  S01  S02  S03  R01  R02  R03  R04    _
PinCount:      0    0    0    0    0    0    0    0    0    0
Popularity:    2    3    3    3    3    2    2    2    2    0
Clock: 0

#requests: 25
#releases: 25
#hits    : 16
#reads   : 9
